src/databricks/labs/ucx/hive_metastore/federation.py

```diff
@@ -0,0 +1,24 @@
+from databricks.labs.blueprint.commands import CommandExecutor
```

Comment:
Do we need a running cluster for this resolution?
Should we use it as part of the assessment?

Reply:
Yes a single user cluster